yarn.lock

```diff
@@ -2865,10 +2865,10 @@ body-parser@1.19.2:
     raw-body "2.4.3"
     type-is "~1.6.18"
 
-bolt11@^1.4.1:
+bolt11-signet@^1.4.1:
```

Comment:
shouldn't this be fixed too? maybe you need to remove and re-install the package?

Reply:
Unfortunately I need to use this package (my custom fork) until it's merged into the original. I made a PR [here](https://github.com/bitcoinjs/bolt11/pull/73), but I can't do anything but wait until that's merged in. For now, though, there's no reason this fork shouldn't be good to use, since it has everything currently in the original + signet capabilities. As soon as that PR is merged and released I will update it in Alby.

Reply:
I mean, in the yarn.lock file it should be `bolt11-signet@1.4.1:` not `bolt11-signet^1.4.1:`

Reply:
To fix this please remove your package, run yarn install, then re-add it in with the fixed version

Reply:
Ah, sorry for misunderstanding. Fixed